src/main/java/Task.java

```diff
@@ -0,0 +1,27 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void Done(){
+        this.isDone = true;
+    }
+
+    public void unDone(){
+        this.isDone = false;
+    }
```

Comment:
Perhaps these methods should be renamed to represent verbs.

```suggestion
    public void markAsDone(){
        this.isDone = true;
    }

    public void unmarkAsDone(){
        this.isDone = false;
    }
```